web/src/views/Login.vue

```diff
@@ -1,26 +1,26 @@
 <template>
-  <main class="flex flex-col w-full h-full justify-center items-center">
-    <Error v-if="errorMessage" class="w-full md:w-3xl">
+  <main class="flex flex-col justify-center items-center w-full h-full">
+    <Error v-if="errorMessage" class="w-full md:max-w-3xl h-96">
       <span class="whitespace-pre">{{ errorMessage }}</span>
-      <span v-if="errorDescription" class="whitespace-pre mt-1">{{ errorDescription }}</span>
+      <span v-if="errorDescription" class="mt-1 whitespace-pre">{{ errorDescription }}</span>
       <a
         v-if="errorUri"
         :href="errorUri"
         target="_blank"
-        class="text-wp-link-100 hover:text-wp-link-200 cursor-pointer mt-1"
+        class="mt-1 text-wp-link-100 hover:text-wp-link-200 cursor-pointer"
       >
         <span>{{ errorUri }}</span>
       </a>
     </Error>
 
     <div
-      class="flex flex-col w-full overflow-hidden bg-wp-background-100 shadow border border-wp-background-400 dark:bg-wp-background-200 md:m-8 md:rounded-md md:flex-row md:w-3xl md:h-sm"
+      class="flex md:flex-row flex-col border-wp-background-400 bg-wp-background-100 dark:bg-wp-background-200 shadow md:m-8 border md:rounded-md w-full md:max-w-3xl md:h-96 overflow-hidden"
```

Comment:
Same here. Why do we need to change it to max-width?

Reply:
Otherwise the Login item is fully horizontally stretched across the page with Tailwind.